docs/SPEC.md

```diff
@@ -1284,6 +1289,73 @@ Examples using known start and stop dates:
 
 [IMPL#659](https://github.com/influxdata/platform/issues/659) Implement intervals function
 
+#### Location support
+
+Intervals have location support for changing offsets by default.
+When a location has been specified either with the `location` option or the `location` parameter, a change in offset is used to either expand or shrink an interval.
+The most common example is daylight savings time switches.
+When daylight savings time starts, the zone offset shifts west, becoming a more negative number.
+When this happens, the hour that was lost does not exist and the current window is shortened to accomodate this change.
+When daylight savings time ends, the zone offset shifts east, becoming a more positive number.
+When this happens, one of the hours happens twice and so the interval becomes longer.
+
+Examples:
+
+    intervals(every:1d, location: "America/Denver")
+    // [2019-03-09T00:00:00-07:00, 2019-03-10T00:00:00-07:00)
+    // [2019-03-10T00:00:00-07:00, 2019-03-11T00:00:00-06:00)
+    // [2019-03-11T00:00:00-06:00, 2019-03-12T00:00:00-06:00)
+    // ... skipped ...
+    // [2019-11-02T00:00:00-06:00, 2019-11-03T00:00:00-06:00)
+    // [2019-11-03T00:00:00-06:00, 2019-11-04T00:00:00-07:00)
+    // [2019-11-04T00:00:00-07:00, 2019-11-05T00:00:00-07:00)
+
+    intervals(every:2h, location: "America/Denver")
+    // [2019-03-10T00:00:00-07:00, 2019-03-10T03:00:00-06:00)
+    // [2019-03-10T03:00:00-06:00, 2019-03-10T04:00:00-06:00)
+    // [2019-03-10T04:00:00-06:00, 2019-03-10T06:00:00-06:00)
+    // ... skipped ...
+    // [2019-11-03T00:00:00-06:00, 2019-11-03T02:00:00-07:00)
+    // [2019-11-03T02:00:00-07:00, 2019-11-03T04:00:00-07:00)
+    // [2019-11-03T03:00:00-07:00, 2019-11-03T06:00:00-07:00)
```

Comment:
Why does this one start at 3 AM -07:00 only one hour after the 2AM -07:00?

Reply:
It's a mistake. I copied a pasted intervals and it was difficult to sort through them.